### PR TITLE
Optionally add member count to Coxir.Struct.Invite.get/2

### PR DESCRIPTION
**Pull request by jb3**
This PR adds the `with_counts?` parameter onto the end of the request to the Discord API.

This allows member counts of a guild to be found using the invite ID, as demonstrated in this image:
screen-835420388

The argument is documented and I have added the missing type-spec to the function.

### CODE REVIEW

**Review by perryprog — APPROVED** [review]

File: lib/coxir/struct/invite.ex [hunk 1]
```diff
@@ -17,9 +17,18 @@ defmodule Coxir.Struct.Invite do
...
+  extra keys, approximate_member_count, the
+  number of members in the server, and
+  approximate_presence_count which is the number
+  of users in the server who are currently online.
   """
-  def get(code) do
-    API.request(:get, "invites/#{code}")
+  @spec get(invite, boolean()) :: map
+
+  def get(code, with_counts \\ false) do
```
Comment (satom99) [inline lib/coxir/struct/invite.ex]: Can the param be renamed to `with_counts?`?

File: lib/coxir/struct/invite.ex [hunk 2]
```diff
@@ -17,9 +17,18 @@ defmodule Coxir.Struct.Invite do
...
+    API.request(:get, "invites/#{code}?with_counts=#{with_counts}")
```
Comment (satom99) [inline lib/coxir/struct/invite.ex]: Can we have the query param set as shown in [here](https://github.com/satom99/coxir/blob/master/lib/coxir/struct/user.ex#L158)? I.e. as  `params: [with_counts: with_counts?]`.

**Review by satom99 — CHANGES_REQUESTED** [review]
Just a few changes. Thanks for the PR!

File: lib/coxir/struct/invite.ex (hunk 1 above)
Comment (jb3) [reply lib/coxir/struct/invite.ex]: Addressed in eb1725b133487b2dee9643ee062104a390a0b36f

File: lib/coxir/struct/invite.ex (hunk 2 above)
Comment (jb3) [reply lib/coxir/struct/invite.ex]: Addressed in eb1725b133487b2dee9643ee062104a390a0b36f

**Review by satom99 — APPROVED** [review]

**Comment by satom99** [comment]
Looking guchie, thanks!

**Pull request merged**